_includes/docs/user-guide/rule-engine-2-0/ce-filter-nodes.md

```diff
@@ -47,38 +48,40 @@ For example, you may use BLE to MQTT gateway and BLE beacons. The Gateway payloa
 {"mac": "7085C2F13DCD", "rssi": -25, "payload": "AABBCC"}
 ```
 
-Let's assume you have different beacon profiles - indoor air quality ("IAQ sensor") and leak sensors ("Leak sensor"). 
+Let's assume you have different beacon profiles - indoor air quality "IAQ sensor" Device Profile and leak sensors "Leak sensor" Device Profile. 
 The below rule chain will change the originator of the message from gateway to device and forward the message to the corresponding rule chain:
 
 ![image](/images/user-guide/rule-engine-2-0/nodes/device-profile-switch-chain.png)
 
 You may [download](https://gist.github.com/ashvayka/f67f9415c625e8a2d12340e18248111f#file-device-profile-switch-example-json) and import the rule chain. 
-Note that the "rule chain" nodes will point to not existing rule chains in your environment.
+Note that the [rule chain nodes](/docs/{{docsPrefix}}user-guide/rule-engine-2-0/flow-nodes/#rule-chain-node) will point to not existing rule chains in your environment.
 
 <br>
 
-## check alarm status
+## alarm status filter {#check-alarm-status}
 
-Checks the [Alarm](/docs/{{docsPrefix}}user-guide/alarms/) status to match one of the specified statuses.
+Filters messages based on the specified [alarm](/docs/{{docsPrefix}}user-guide/alarms/) statuses.
 
 **Configuration**
 
- * Alarm status filter - Contains list of alarms statuses. 
-  Available statuses: "Active Acknowledged", "Active Unacknowledged", "Cleared Acknowledged", "Cleared Unacknowledged".
+ * **Alarm status** - selection form with the statuses you want to filter by. Available statuses: "Active Acknowledged", "Active Unacknowledged", "Cleared Acknowledged", "Cleared Unacknowledged".
 
 ![image](/images/user-guide/rule-engine-2-0/nodes/check-alarm-status-configuration.png)
 
 **Output**
 
-Output connection types: "True" or "False".
+ * **True**: if the message matches any of the selected alarm statuses.
+ * **False**: if the message does not match any of the selected alarm statuses.
 
-**Example**
+**Usage example**
 
-The rule chain below will check that the acknowledged alarm is still active or already cleared.
+Consider a scenario where you want to process alarms that are currently active. 
+You can configure the rule node to filter for "Active Unacknowledged" and "Active Acknowledged" statuses. 
+This setup ensures that only alarms which are currently active, whether they have been acknowledged or not, are processed further.
 
 ![image](/images/user-guide/rule-engine-2-0/nodes/check-alarm-status-chain.png)
 
-You may [download](https://gist.github.com/ashvayka/f67f9415c625e8a2d12340e18248111f#file-check-alarm-status-example-json) and import the rule chain.
+You may [download](https://gist.github.com/ShvaykaD/dce641880a78013a273f8f8c82fa7f1e#file-alarm-status-filter-example-json) and import the rule chain.
```

Comment:
@ashvayka should we merge this to your original gists, or we can leave this as it is with the version from my fork?